src/main/kotlin/com/autonomousapps/advice/Advice.kt

```diff
@@ -87,13 +98,23 @@ data class Advice(
   fun isAdd() = fromConfiguration == null && !isCompileOnly()
 
   /**
-   * An advice is "remove-advice" if it is declared and not used, AND is not `compileOnly`.
+   * An advice is "remove-advice" if it is declared and not used, AND is not `compileOnly`,
+   * AND is not `processor`.
    */
-  fun isRemove() = toConfiguration == null && !isCompileOnly()
+  fun isRemove() = toConfiguration == null && !isCompileOnly() &&!isProcessor()
 
   /**
    * An advice is "change-advice" if it is declared and used (but is on the wrong configuration),
    * AND is not `compileOnly`.
    */
   fun isChange() = fromConfiguration != null && toConfiguration != null && !isCompileOnly()
+
+  /**
+   * An advice is "processors-advice" if it is declared on a k/apt
+   * or annotationProcessor configuration.
+   */
+  fun isProcessor() = fromConfiguration != null &&
+    toConfiguration == null &&
+    (fromConfiguration.endsWith("apt", ignoreCase = true)
```

Comment:
I appreciate this attempt to be more general (apt vs kapt), but actually we only check for kapt currently. I've made no attempt to be compatible with the old `apt`. And I'm still not convinced we need to differentiate processor-advice from remove-advice like this.